spec/teacher_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './teacher'
+require './person'
+
+describe Teacher do
+  context 'check for correct values passed through teacher class ' do
+    new_teacher = Teacher.new(55, 'English', 'Mark')
+    it 'check for an instance of teacher age' do
+      expect(new_teacher.age).to eql(55)
+    end
+
+    it 'check for an instance of teacher name' do
+      expect(new_teacher.name).to eql('Mark')
+    end
+
+    it 'check for an instance of teacher specialization' do
+      expect(new_teacher.specialization).to eql('English')
+    end
+  end
+end
```

Comment:
- [ ] As require for the project to add unit tests for all the methods in all the classes, kindly make sure you also implement unit tests for the `can_use_services?` method from the `Teacher` class.